about.html

```diff
@@ -0,0 +1,104 @@
+            <div class="hamburger">
+                <a href="#"><img id="menu" src="./images/menu.png" alt="Hamburger"></a>
+            </div>
+            <nav class="mobileNav">
+                <ul id="nav">
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">News</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                  </ul>
+            </nav>
+        </nav>
+    </header>
+
+    <section class="guest">
+        <div class="commons">
+            <span class="she">SheCodes women conference</span>
+            <h2>GUEST SPEAKER ON TECHNICAL WRITING</h2>
+            <P>SheCodes brings together an experienced group of female sofware developers who talk about there journey in tech. Highlighting all the key points that help beginners get more comfortable starting out. They share usefull resources that act as pointers to help newbies.
+                Join us as we talk about our journey and experiences. </P>
+            <span class="bottom">
+                Please contact us via email if you have any questions or suggestions
+            </span>
+            <h3 class="mail"><a href="acholahlynette111@gmail.com">shecodesglobal@gmail.com</a></h3>
+        </div>
+    </section>
+    <section class="sheCodes">
+        <div class="global">
+            <h3>SheCodes global goal for 2022</h3>
+            <hr class="line">
+        </div>
+        <div class="sheCode">
+            <p>sheCodes aim is to reach all the women in tech from all over the world and build a network of female developers. sheCodes aim is to reach all the women in tech from all over the world and build a network of female developers</p>
+            <span>SheCodesGlobal</span>
+        </div>
+        
+    </section>
+    <hr class="alone">
+    <section class="summit">
+        <div class="past">
+            <h3>See the past sheCode's meetup events</h3>
+            <hr class="line">
+            <p>View the recordings from the past few sessions to know what to expect from our speakers</p>
+        </div>
+        <div class="activities">
+            <div class="activityContainer">
+                <div class="text1">
+                    <span class="aa">2020</span>
+                    <span class="bb">First keynote speaker</span>
+                </div>
+                <img src="./images/key5.jpg" alt="speaker1">
+            </div>
+            <div class="activityContainer">
+                <div class="text2">
+                    <span class="aa">2021</span>
+                    <span class="bb">Second keynote speaker</span>
+                </div>
+                <img src="./images/key5.jpg" alt="speaker2">
+            </div>
+        </div>
+    </section>
+
+    <footer class="icons2">
+        <p>Copyright © 2021 All rights reserved | sheCodes</p>
+    </footer>
```

Comment:
- [ ] **Required** In mobile version ,The `about page ` `footer section` is same as `home` page footer. Right now there is text with the color white present there, Kindly fix that.